frc_characterization/drive_characterization/data_logger.py

```diff
@@ -282,24 +295,24 @@ def runTest(self, name, initial_speed, ramp, finished, rotate=None):
             with self.lock:
                 self.lock.wait_for(lambda: self.mode == "auto")
 
-            data = self.wait_for_stationary()
-            if data is not None:
-                if data in ("connected", "disconnected"):
-                    self.STATE.postTask(
-                        lambda: messagebox.showerror(
-                            "Error!", "NT disconnected", parent=self.STATE.mainGUI
-                        )
-                    )
-                    return
-                else:
-                    self.STATE.postTask(
-                        lambda: messagebox.showerror(
-                            "Error!",
-                            "Robot exited autonomous mode before data could be sent?",
-                            parent=self.STATE.mainGUI,
-                        )
-                    )
-                    return
+            #            data = self.wait_for_stationary()
+            #            if data is not None:
+            #                if data in ("connected", "disconnected"):
+            #                    self.STATE.postTask(
+            #                        lambda: messagebox.showerror(
+            #                            "Error!", "NT disconnected", parent=self.STATE.mainGUI
+            #                        )
+            #                    )
+            #                    return
+            #                else:
+            #                    self.STATE.postTask(
+            #                        lambda: messagebox.showerror(
+            #                            "Error!",
+            #                            "Robot exited autonomous mode before data could be sent?",
+            #                            parent=self.STATE.mainGUI,
+            #                        )
+            #                    )
+            #                    return
```

Comment:
Why was this commented out? Also, unused/commented out code should be removed.

Reply:
`data = self.wait_for_stationary()` is an unnecessary precaution that requires the robot to send network tables data during the autonomous period. Removing this allows us to only send data once the robot enters disabled.

Reply:
That makes sense; you should remove it outright. We have version control so we’ll be able to go back and look at the code if we need to.